src/main/java/xyz/vegaone/easytrackingv2/domain/UserEntity.java

```diff
@@ -16,6 +17,12 @@
     @Column(name = "email")
     private String email;
 
+    @ManyToMany(cascade = {CascadeType.ALL}, fetch = FetchType.EAGER)
```

Comment:
Using  `CascadeType.ALL` will produce unwanted effects. e.g. deleting the user will delete in cascade all the projects associated with the user. This is not something we want. An employee might have left the company and so we will remove his/her user, but the project should go on with the other users.
One way to address this issue is to change the entity that manages the relationship from `User` to `Project` (put the CascadeType and all that in the `ProjectEntity`). But that will not be enough since removing a project will also remove the user.
Another way is to specify which operations should be cascaded. Instead of `CascadeType.ALL` we could have `cascade = {CascadeType.PERSIST, CascadeType.MERGE}` and so on.  `CascadeType.REMOVE` is rarely used. Please check which types of `Cascade` are available and which make sense to use here.